core/tx_pool.go

```diff
@@ -526,10 +527,12 @@ func (pool *TxPool) stats() (int, int) {
 // StatsWithMinBaseFee retrieves the current pool stats, namely the number of pending and the
 // number of queued (non-executable) transactions greater equal minBaseFee.
 func (pool *TxPool) StatsWithMinBaseFee(minBaseFee *big.Int) (int, int) {
+	statsStart := time.Now()
```

Comment:
Should we log the time _before_ we acquire the log? If it's like this then the `statsWithMinBaseFeeTimer` does include both the runtime of this operation and the waiting time on the lock no? Is this what we want here?

Reply:
make sense